src/raft_server.c

```diff
@@ -991,6 +999,7 @@ int raft_recv_requestvote(raft_server_t *me,
              req->last_log_term);
 
     resp->prevote = req->prevote;
+    req->prevote ? me->stats.requestvote_prevote_req_received++ : me->stats.requestvote_req_received++;
```

Comment:
```suggestion
    req->prevote ? me->stats.requestvote_prevote_req_received++ : me->stats.requestvote_req_received++;

    resp->prevote = req->prevote;
```

Let's move this out of `resp` assignments to make it easier to read.

Reply:
ok.